api/clients/config.go

```diff
@@ -8,22 +8,32 @@ import (
 )
 
 type EigenDAClientConfig struct {
-	// RPC is the HTTP provider URL for the Data Availability node.
+	// RPC is the HTTP provider URL for the EigenDA Disperser
 	RPC string
 
-	// The total amount of time that the client will spend waiting for EigenDA to confirm a blob
+	// Timeout used when making dispersals to the EigenDA Disperser
```

Comment:
This reads like it's waiting for the gRPC call to finish DisperseBlob(), but the old comment says it's waiting for confirmation. Is this intended change?

Reply:
I think it is indeed just the timeout for dispersal call, https://github.com/Layr-Labs/eigenda/blob/master/api/clients/disperser_client.go#L148

Reply:
The old comment was not accurate... not sure if/when it was changed.